mne/report.py

```diff
@@ -314,26 +312,31 @@ def _update_html(html, report_fname, report_sectionlabel):
 
     return htmls, report_fnames, report_sectionlabels
 
+
 ###############################################################################
 # IMAGE FUNCTIONS
 
+def _figure_agg(**kwargs):
+    from matplotlib.figure import Figure
+    from matplotlib.backends.backend_agg import FigureCanvasAgg as FigureCanvas
+    fig = Figure(**kwargs)
+    FigureCanvas(fig)
+    return fig
+
 
 def _build_image_png(data, cmap='gray'):
     """Build an image encoded in base64."""
     import matplotlib.pyplot as plt
-    from matplotlib.figure import Figure
-    from matplotlib.backends.backend_agg import FigureCanvasAgg as FigureCanvas
 
     figsize = data.shape[::-1]
     if figsize[0] == 1:
         figsize = tuple(figsize[1:])
         data = data[:, :, 0]
-    fig = Figure(figsize=figsize, dpi=1.0, frameon=False)
-    FigureCanvas(fig)
+    fig = _figure_agg(figsize=figsize, dpi=92, frameon=False)
     cmap = getattr(plt.cm, cmap, plt.cm.gray)
     fig.figimage(data, cmap=cmap)
     output = BytesIO()
-    fig.savefig(output, dpi=1.0, format='png')
+    fig.savefig(output, dpi=92, format='png')
```

Comment:
why did you change this?

Reply:
1.0 was a strange choice and seemed like a bug

Reply:
it does not expand the size of a regular report? these can easily become huge. Maybe it was the tradoff?